README.md

```diff
@@ -0,0 +1,158 @@
+## status-go-sdk
+This package provides an easy way to interact with [status](https://status.im) messaging system.
+This package allows you create or  application interact with [status](https://status.im) messaging system.
+
+## Motivation
+This library was initially created to have a low-dependency way to interact with status messaging system.
+
+## Build status
+
+[![Build Status](https://travis-ci.org/akashnimare/foco.svg?branch=master)](https://travis-ci.org/akashnimare/foco)
+
+## Features
+This project is under heavy development process, here is a list of supported features:
+
+**Status messaging basic interaction**
+- [x] Setup a new connection
+- [x] Close a specific connection
+- [x] Sign up on the platform
+- [x] Login as a specific account
+- [ ] Documented API for basic sdk interaction
+
+**Public channels interaction**
+- [x] Join public channels
+- [x] Publish messages on public channels
+- [x] Subscribe to public channels
+- [x] Unsubscribe from a public channel
+- [ ] Documented API for public channels interaction
+- [x] Working examples for public channel interaction
+
+**Contact management**
+- [ ] Invite new contact
+- [ ] Accept a contact request
+
+**1 to 1 messages interaction**
+- [ ] Send 1 to 1 conversation
+- [ ] Subscribe to 1 to 1 conversation
+- [ ] Unsubscribe from a 1 to 1 conversation
+- [ ] Documented API for 1 to 1 conversation
+- [ ] Working examples for 1 to 1 conversations
+
+**Private groups interaction**
+- [ ] Publish messages on private groups
+- [ ] Subscribe to private groups
+- [ ] Unsubscribe from a private groups
+- [ ] Documented API for private groups interaction
+- [ ] Working examples for private groups interaction
+
+## Code Example
+You'll find some code examples on the [examples folder](/examples)
+
+## Installation
+status-go-sdk is a lightweight dependency package, that means by now it does not have any external dependency. You can install it by running:
+```
+go get github.com/status-im/status-go
+```
+
+## API Reference
+TBD
```

Comment:
isn't this usually covered by godoc?

Reply:
yep, since this is on hold i haven't had time to adapt this to be used by godoc, and we neither have a proper CI in place, I'll try to open some issues to manage this problems.